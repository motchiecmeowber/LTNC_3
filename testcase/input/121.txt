BEGIN
INSERT x number
ASSIGN x 9
END
INSERT y string
ASSIGN y 2a
PRINT