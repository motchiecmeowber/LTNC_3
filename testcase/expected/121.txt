Invalid: ASSIGN y 2a